### ISSUE TITLE
Is TAPLoss only used to fine-tune a model?

### ISSUE THREAD

**Issue by SherryYu33** [YunyangZeng/TAPLoss#7]
Hello.
I'm training a new model. Can I add TAPLoss to my loss function? Or is it only used to fine-tune a model?
Thank you!

**Comment by muqiaoy** [comment]
Yes, it can be used to train a new model as an auxiliary loss function in addition to the original one.

**Comment by YunyangZeng** [comment]
You can use a scheduler to gradually increase the weight of the Taploss. A high Taploss weight at the early stage might cause your enhancement model to diverge. The reason for this is that the TAP estimator is trained with clean speech, but in early stage your enhanced speech is still very noisy, thus the estimated acoustics are not accurate and might cause the training to be less stable.

**Comment by SherryYu33** [comment]
Thank you
